README.md

```diff
@@ -33,20 +24,30 @@ In case of error try
 service netfilter-persistent save  
 service netfilter-persistent reload
 ```
-To test glutton
+Now, install dependencies using `glide`.
 ```
-mkdir -p /etc/glutton  
-mkdir -p /var/log/glutton
-go get github.com/google/gopacket
-go get pkg.re/essentialkaos/z7.v2
-cp $GOPATH/src/github.com/mushorg/glutton/config/ports.yml /etc/glutton
-go run $GOPATH/src/github.com/mushorg/glutton/server/glutton_server.go -log /tmp/glutton.log
+curl https://glide.sh/get | sh
+glide install
+glide update
 ```
-To make glutton start on boot using upstart
+Download and setup glutton,
 ```
-cd $GOPATH/src/github.com/mushorg/glutton/glutton
-go install
-cp $GOPATH/src/github.com/mushorg/glutton/scripts/glutton.conf /etc/init
+go get github.com/mushorg/glutton
+cd $GOPATH/src/github.com/mushorg/glutton
+go install app/server.go
+mkdir /etc/glutton
+cp $GOPATH/src/github.com/mushorg/glutton/rules/rules.yaml /etc/glutton
+```
+To run glutton,
+```
+cd $GOPATH/bin
+./app
```

Comment:
I don't think it makes a difference. I wouldn't spend too much time thinking about it. It's also not related to the issue you want to solve with this pull request.